cmd/soroban-rpc/internal/test/metrics_test.go

```diff
@@ -27,6 +33,19 @@ func TestMetrics(t *testing.T) {
 	require.Contains(t, metrics, buildMetric)
 }
 
+func TestLogMetrics(t *testing.T) {
```

Comment:
this test is not relevant to soroban-rpc because it is testing the correctness of the logmetrics package which is implemented in the go monorepo. Also, I think any functionality exercised in this test is already duplicated in the existing logmetrics test:

https://github.com/stellar/go/blob/master/support/logmetrics/main_test.go#L16

I think you should remove TestLogMetrics() entirely and, instead, you can add an extra check in `TestMetrics()` which verifies that the log metrics are getting included in the metrics response